index.js

```diff
@@ -70,21 +71,27 @@ function lint(textEditor) {
 			};
 		}
 
+		// taken from linter-eslint
+		let range;
+		const msgLine = x.line - 1;
+		if (typeof x.endColumn === 'number' && typeof x.endLine === 'number') {
+			const msgCol = Math.max(0, x.column - 1);
+			range = [[msgLine, msgCol], [x.endLine - 1, x.endColumn - 1]];
+		} else if (typeof x.line === 'number' && typeof x.column === 'number') {
+			// We want msgCol to remain undefined if it was initially so
+			// `rangeFromLineNumber` will give us a range over the entire line
+			const msgCol = typeof x.column === 'undefined' ? x.column : x.column - 1;
+			range = rangeFromLineNumber(textEditor, msgLine, msgCol);
```

Comment:
Note that if an invalid point is provided this will `throw` an error, you may want to catch and handle this somehow.

ESLint has had a variety of issues with rules in the past, many fixed due to this catching the bugs and users reporting them 😛.